scripts/syncSettings.py

```diff
@@ -22,113 +22,135 @@
 }
 
-def fastBlocksSettings(configuration, apiUrl, blockReduced, multiplierRequirement):
+def fastBlocksSettings(configuration, apiUrl, blockReduced, multiplierRequirement, isPoS):
     if "etherscan" in apiUrl:
-        latestBlock = int(json.loads(subprocess.getoutput(f'curl --silent "https://{apiUrl}/api?module=proxy&action=eth_blockNumber&apikey={key}"'))['result'],16)
+        latestBlock = int(json.loads(subprocess.getoutput(
+            f'curl --silent "https://{apiUrl}/api?module=proxy&action=eth_blockNumber&apikey={key}"'))['result'], 16)
     else:
-        data = '{"id":0,"jsonrpc":"2.0","method": "eth_blockNumber","params": []}'
-
-        response = requests.post(apiUrl, headers=headers, data=data).text
+        data_req = '{"id":0,"jsonrpc":"2.0","method": "eth_blockNumber","params": []}'
+        response = requests.post(apiUrl, headers=headers, data=data_req).text
         latestBlock = int(json.loads(response)['result'], 16)
 
     baseBlock = latestBlock - blockReduced
     baseBlock = baseBlock - baseBlock % multiplierRequirement
 
     if "etherscan" in apiUrl:
-        pivot = json.loads(subprocess.getoutput(f'curl --silent "https://{apiUrl}/api?module=proxy&action=eth_getBlockByNumber&tag={hex(baseBlock)}&boolean=true&apikey={key}"'))
+        pivot = json.loads(subprocess.getoutput(
+            f'curl --silent "https://{apiUrl}/api?module=proxy&action=eth_getBlockByNumber&tag={hex(baseBlock)}&boolean=true&apikey={key}"'))
     else:
-        data = '{"id":0,"jsonrpc":"2.0","method": "eth_getBlockByNumber","params": ["' +str(hex(baseBlock))+ '", false]}'
-        pivot = json.loads(requests.post(apiUrl, headers=headers, data=data).text)
+        data_req = '{"id":0,"jsonrpc":"2.0","method": "eth_getBlockByNumber","params": ["' + str(hex(baseBlock)) + '", false]}'
+        pivot = json.loads(requests.post(apiUrl, headers=headers, data=data_req).text)
 
     pivotHash = pivot['result']['hash']
     pivotTotalDifficulty = int(pivot['result'].get('totalDifficulty', '0x0'), 16)
-    print(configuration + 'LatestBlock: ' + str(latestBlock))
-    print(configuration + 'PivotNumber: ' + str(baseBlock))
-    print(configuration + 'PivotHash: ' + str(pivotHash))
-    print(configuration + 'PivotTotalDifficulty: ' + str(pivotTotalDifficulty))
-    data = {}
+
+    print(configuration + ' LatestBlock: ' + str(latestBlock))
+    print(configuration + ' PivotNumber: ' + str(baseBlock))
+    print(configuration + ' PivotHash: ' + str(pivotHash))
+    if not isPoS:
+      print(configuration + ' PivotTotalDifficulty: ' + str(pivotTotalDifficulty))
+
     with open(f'{configsPath}/{configuration}.json', 'r') as mainnetCfg:
         data = json.load(mainnetCfg)
-        data['Sync']['PivotNumber'] = baseBlock
-        data['Sync']['PivotHash'] = pivotHash
+
+    data['Sync']['PivotNumber'] = baseBlock
+    data['Sync']['PivotHash'] = pivotHash
+  
+    if not isPoS:
         data['Sync']['PivotTotalDifficulty'] = str(pivotTotalDifficulty)
```

Comment:
So it does not set it now? And set to null?

Reply:
It will simply not change it json - so keep current value for PoS chains as it is static since merge.